include/rssfeed.h

```diff
@@ -153,6 +153,11 @@ class RssFeed : public Matchable {
 		return rssurl_.substr(0, 6) == "query:";
 	}
 
+	bool is_search_feed() const
+	{
+		return rssurl_.substr(0, 7) == "search:";
```

Comment:
`rssurl_` comes from the urls file, i.e. from the user. Let's not re-purpose that. Instead, I suggest adding a new private `bool`, plus a setter method `set_is_search_feed`.

Instead of calling that setter in `ItemListFormAction::qna_start_search`, it's better to call it where the feed is created. At the bottom of `ItemListFormAction::qna_start_search` you can see that the search results are always put into a `search_dummy_feed`; the same dummy exists in `FeedListFormAction` as well. The dummy never changes, so we can call `set_is_search_feed` once, in the constructor of `ItemListFormAction`/`FeedListFormAction`.